### PR TITLE
List structure #PR

### PR DESCRIPTION
**Pull request by Sonikak004**
### In this branch I have : 👍 

- Set up a new project with webpack.
- Created an index.html file with HTML markup. 
- Created an index.js file, in which Each task object contains three keys:
    - description [string].
    - completed [bool].
    - index: [number].
- On page load renders the dynamically created list of tasks in the dedicated placeholder.
- Created a style.css and set rules for the To Do List. 
- CSS is loaded by Webpack Style/CSS Loader.
- Fix linter errors

### CODE REVIEW

**Review by CollinsTatang — COMMENTED** [review]
Hi @Sonikak004 , 

While you made a great effort in this project. Unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, without a  README file.

- [x] Please make sure all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.
- [x] Please make sure that you add the section **Future Features**, including 1-3 potential features to be added.
- [x] Please remove the Microverse logo from your README file


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
```
Comment (souad988) [inline README.md]: Kindly add the missing sections and links as requeted by the previous reviewer only FAQ is optional

**Review by souad988 — COMMENTED** [review]
Hi @Sonikak004  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has README file missing some sections from the template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/task.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  tasks.push(newTask);
+  renderTasks();
+  taskInput.value = '';
+};
+
+export const deleteAllTasks = () => {
+  tasks = [];
+  renderTasks();
+};
```
Comment (roshan-bajgain) [inline src/task.js]: - [x] In this project you need to render the list using Javascript as the add and remove functionality will be implemented in a further project that is next project.
- [x] In the requirement you should create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element) and using index.js file and set an array of some simple to do tasks (array of objects). For example: add more hardcoded to-do list
`const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 1,
  },
];
`
then Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- [ ] The data should be populated according to the index value.

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+.completed {
+  text-decoration: line-through;
+}
+
+.wrapper {
+  display: flex;
+  height: 97vh;
+  align-content: center;
+  flex-wrap: wrap;
+}
```
Comment (roshan-bajgain) [inline src/style.css]: - [x] CSS should be created to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Please style the app as in the design.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @Sonikak004, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- GitHub flow is followed. ✔️
- Good Pull request description and title ✔️ 


## Required Changes ♻️
- [x] In the next project you will implement the add and remove so for now you can have just hardcoded list as I told on inline comment.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @Sonikak004, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights 🎯
- Descriptive PR title and summary ✅
- Follows Github flow ✅
- Good commit messages ✅

## Optional suggestions 🪶

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**